### PR TITLE
Add priors to git repo and adjust container build

### PR DESCRIPTION
**Pull request by damonbayer**
Closes https://github.com/CDCgov/pyrenew-hew/issues/238

### CODE REVIEW

**Comment by damonbayer** [comment]
A bit unclear what's going on with the docker build.

It succeeded [here](https://github.com/CDCgov/pyrenew-hew/actions/runs/12363032818/job/34503586601?pr=251), immediately after I introduced the `COPY --exclude`.

Then it [started complaining about the exclude flag](https://github.com/CDCgov/pyrenew-hew/actions/runs/12363602411/job/34505253533?pr=251)

Now it is [succeeding](https://github.com/CDCgov/pyrenew-hew/actions/runs/12363685587/job/34505506839?pr=251) after I added `# syntax=docker/dockerfile:1.7-labs`

**Comment by damonbayer** [comment]
Seems like the container build is reliably working now.

File: pipelines/forecast_state.py [hunk 1]
```diff
@@ -21,21 +21,48 @@
...
-    metadata = {
+    new_metadata = {
         "branch_name": branch_name,
         "commit_sha": commit_sha,
     }
 
+    metadata.update(new_metadata)
+
+    with open(metadata_file, "w") as file:
+        yaml.dump(metadata, file)
```
Comment (dylanhmorris) [inline pipelines/forecast_state.py:47]: toml rather than yaml metadata, for consistency with toml elsewhere?

**Review by dylanhmorris — COMMENTED** [review]
One question @damonbayer

File: pipelines/forecast_state.py (hunk 1 above)
Comment (damonbayer) [reply pipelines/forecast_state.py:47]: Not sure what you mean by "toml elsewhere," but I can change it.

Comment (damonbayer) [reply pipelines/forecast_state.py]: https://github.com/CDCgov/pyrenew-hew/pull/251/commits/a31a7e06acbbb6640f5a43c047871faa9f46318d

Comment (dylanhmorris) [reply pipelines/forecast_state.py]: Oh right we used to have some and then moved back to using `.py` files to specify priors

File: pipelines/forecast_state.py [hunk 2]
```diff
@@ -20,24 +21,52 @@
...
         "commit_sha": commit_sha,
     }
 
-    with open(metadata_file, "w") as file:
-        yaml.dump(metadata, file)
+    metadata.update(new_metadata)
+
+    metadata_file.parent.mkdir(parents=True, exist_ok=True)
+    with open(metadata_file, "wb") as file:
+        tomli_w.dump(metadata, file)
```
Comment (dylanhmorris) [inline pipelines/forecast_state.py:49]: Typo (I think?)

```suggestion
        tomllib.dump(metadata, file)
```

**Pull request merged**